prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -5,18 +5,9 @@ package gov.cdc.prime.router.cli
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.groups.mutuallyExclusiveOptions
 import com.github.ajalt.clikt.parameters.groups.single
-import com.github.ajalt.clikt.parameters.options.convert
-import com.github.ajalt.clikt.parameters.options.flag
-import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.*
 import com.github.ajalt.clikt.parameters.types.int
-import gov.cdc.prime.router.DocumentationFactory
-import gov.cdc.prime.router.FakeReport
-import gov.cdc.prime.router.FileSource
-import gov.cdc.prime.router.Metadata
-import gov.cdc.prime.router.OrganizationService
-import gov.cdc.prime.router.Report
-import gov.cdc.prime.router.Schema
-import gov.cdc.prime.router.Translator
+import gov.cdc.prime.router.*
```

Comment:
There is a Kotlin style guide violation against using wildcards.

Reply:
I will fix this.